<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="Live_Graph" name="graph" native="true"/>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="connectionsStatusDisplay_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="SMU_Manual_Controller" name="smuManualController_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Function_Generator_Controller" name="functionGenerator_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Temperature_Hold" name="temperatureHold_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="turnOnDSO_pushButton">
         <property name="text">
          <string>Turn On DSO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settings_pushButton">
         <property name="text">
          <string>Configuration</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>147</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="takeMeasurement_pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Take Single Measurement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Temperature Range (in K)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="start_label_2">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startTemp_lineEdit">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>90</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="end_label_2">
              <property name="text">
               <string>End</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="endTemp_lineEdit">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>300</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="step_label_2">
              <property name="text">
               <string>Step</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="stepTemp_lineEdit">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="currentTemp_label">
              <property name="text">
               <string>Current Temperature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="currentTemp_lineEdit">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="currentTemp_label_3">
              <property name="text">
               <string>Heater Output</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="outputPower_lineEdit">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="takeMeasurementSweep_pushButton">
         <property name="text">
          <string>Measurement Sweep</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="descriptionFile_label">
         <property name="text">
          <string>Measurement Description File</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="descriptionFilePath_lineEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadDevicesFile_pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Transimpedance Gain</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="transimpedance100_checkBox">
            <property name="text">
             <string>10²</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="transimpedance1000_checkBox">
            <property name="text">
             <string>10³</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="transimpedance10000_checkBox">
            <property name="text">
             <string>10⁴</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="transimpedance100000_checkBox">
            <property name="text">
             <string>10⁵</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Save</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="user_label">
            <property name="text">
             <string>User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="user_lineEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sampleName_label">
            <property name="text">
             <string>Sample Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sampleName_lineEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveToDatabase_pushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save To Database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outputToFile_pushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Output To File</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Live_Graph</class>
   <extends>QWidget</extends>
   <header>MPL_Shared.Live_Graph.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SMU_Manual_Controller</class>
   <extends>QWidget</extends>
   <header>MPL_Shared.SMU_Manual_Controller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Function_Generator_Controller</class>
   <extends>QWidget</extends>
   <header>MPL_Shared.Function_Generator_Controller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Temperature_Hold</class>
   <extends>QWidget</extends>
   <header>MPL_Shared.Temperature_Hold.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
